ports/esp32/src/device.c

```diff
@@ -844,6 +844,16 @@ void Device_getCurrentDateTime(BACNET_DATE_TIME *DateTime)
     DateTime->time = Local_Time;
 }
 
+void Device_getCurrentTime(BACNET_TIME *Time)
+{
+    Update_Current_Time();
+
+    Time->hour = Local_Time.hour;
+    Time->min = Local_Time.min;
+    Time->sec = Local_Time.sec;
+    Time->hundredths = Local_Time.hundredths;
+}
```

Comment:
Is this duplicated code required? (Duplicate from apps/piface/device.c) line 843